src/pkg/fault/fault.go

```diff
@@ -208,6 +208,32 @@ func (e *Bus) Errors() *Errors {
 	}
 }
 
+// ErrorEntries returns the items that failed along with other
+// recoverable errors
+func (e *Bus) ErrorEntries() ([]Item, []error) {
```

Comment:
Nit: this name is a little too generic.  From a certain perspective, all errors in the fault are entries.
```suggestion
func (e *Bus) ItemsAndRecovered() ([]Item, []error) {
```